crates/swc_ecma_transforms_proposal/tests/decorators/issue-7358/2/output.js

```diff
@@ -0,0 +1,42 @@
+export function test() {
+    var _dec, _initClass, _dec1, _dec2, _initProto;
+    let _Foo;
+    _dec = decorate(), _dec1 = decorate(), _dec2 = decorate();
```

Comment:
I added a test for this because I think in my last PR this expression was accidentally above the `var` decls, which is not a big deal, but not ideal.